foodsaving/groups/stats.py

```diff
@@ -34,6 +34,24 @@ def group_activity(group):
     }])
 
 
+def group_summary_email(group, recipient_count):
+    write_points([{
+        'measurement': 'karrot.email.group_summary',
+        'tags': {
+            'group': str(group.id)
+        },
+        'fields': {'value': 1, 'recipient_count': recipient_count},
+    }])
+
+
+def periodic_task(name):
```

Comment:
Naming seems quite generic, but fine for now

Reply:
Yes, I think it would be used across apps, so belongs somewhere more general, but didn't find the place yet...